src/pwa/components/Menu/MenuButton.js

```diff
@@ -4,16 +4,22 @@ import { inject } from 'mobx-react';
 import IconMenu from 'react-icons/lib/md/menu';
 import { Container } from '../../../shared/styled/Menu/MenuButton';
 
-const MenuButton = ({ menuHasOpen }) => (
-  <Container onClick={menuHasOpen}>
+const MenuButton = ({ menuHasOpen, sendEvent, component }) => (
+  <Container onClick={() => {
```

Comment:
We can try [withHandlers](https://github.com/acdlite/recompose/blob/master/docs/API.md#withhandlers) here 👍